nominal/nominal.py

```diff
@@ -339,3 +348,41 @@ def _get_start_end_timestamp_csv_file(
         IntegralNanosecondsUTC(start.to_datetime64().astype(int)),
         IntegralNanosecondsUTC(end.to_datetime64().astype(int)),
     )
+
+def create_checklist(
+    assignee_email: str,
+    title: str,
+    checks: Sequence[CreateCheck],
+    checklist_variables: Sequence[CreateChecklistVariable],
+    default_ref_name: str | None = None,
+    commit_message: str | None = None,
+    description: str | None = None,
+    properties: Mapping[str, str] | None = None,
+    labels: Sequence[str] = (),
```

Comment:
this function has a ton of params which will make it more cumbersome to use and document; unless we need everything here I'd prefer to make the signature smaller, and provide the .update() method to add props and labels like we do with the other functions

Reply:
Discussed offline: switching to a builder